jsx/Panel.js

```diff
@@ -46,21 +46,23 @@ class Panel extends Component {
     // Add panel header, if title is set
     const panelHeading = this.props.title ? (
       <div
-        className="panel-heading"
+        className='panel-heading'
         onClick={this.toggleCollapsed}
-        data-toggle="collapse"
+        data-toggle={this.props.collapsing ? 'collapse' : ''}
         data-target={'#' + this.props.id}
-        style={{cursor: 'pointer'}}
+        style={this.props.collapsing ?
+          {cursor: 'pointer'} : {cursor: 'default'}
+        }
       >
-        {this.props.title}
-        <span className={glyphClass}></span>
+        <h3 className={'panel-title'}>{this.props.title}</h3>
+        {this.props.collapsing ? <span className={glyphClass}/> : ''}
```

Comment:
rendering ' ' if collapsing is false makes me feel bad :( could we put all this into a `const` and render that const instead?

Reply:
@zaliqarosli done and made it null if false